perfschema/statement.go

```diff
@@ -160,7 +160,10 @@ func (ps *perfSchema) StartStatement(sql string, connID uint64, callerName EnumC
 			return nil
 		}
 		source = fmt.Sprintf("%s:%d", fileName, fileLine)
+
+		callerLock.Lock()
```

Comment:
Yes you are right, we should lock map even if we read it in golang.

 lock is ok because write is only once.